pages/linux/chrt.md

```diff
@@ -17,4 +17,4 @@
 
 - Set the scheduling policy for a process:
 
-`chrt --pid {{PID}} --{{deadline|idle|batch|rr|fifo|other}}`
+`chrt --{{deadline|idle|batch|rr|fifo|other}} --pid {{PID}}`
```

Comment:
This doesn't work either - the priority has to be specified to set the scheduling policy.
```
$ chrt --help
Show or change the real-time scheduling attributes of a process.

Set policy:
 chrt [options] <priority> <command> [<arg>...]
 chrt [options] --pid <priority> <pid>
```

---

```
$ chrt --idle --pid {{PID}}
pid {{PID}}'s current scheduling policy: SCHED_OTHER
pid {{PID}}'s current scheduling priority: 0
```
The scheduling policy is still SCHED_OTHER, even though `--idle` was used.

With the priority:
```
$ chrt --idle --pid 0 {{PID}}
(this command doesn't output anything)
$ chrt --pid {{PID}}
pid {{PID}}'s current scheduling policy: SCHED_IDLE
pid {{PID}}'s current scheduling priority: 0
```

```suggestion
`chrt --{{deadline|idle|batch|rr|fifo|other}} --pid {{priority}} {{PID}}`
```